### PR TITLE
Release/v3.0

### PR DESCRIPTION
**Pull request by ggsdc**
Solves #80 and #161

### CODE REVIEW

**Review by ampuerin — APPROVED** [review]
great work!

File: cornflow/commands/permissions.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        not in permissions_registered
+    ]
+
+    if len(permissions_to_register) > 0:
+        db.session.bulk_save_objects(permissions_to_register)
+
+    try:
+        db.session.commit()
+    except IntegrityError:
+        db.session.commit()
```
[the remark this answers is not in the document]
Comment (ggsdc) [reply cornflow/commands/permissions.py:58]: You are completely right!

File: cornflow/endpoints/schemas.py [hunk 2]
```diff
@@ -29,13 +32,12 @@ def get(self):
         :return: A dictionary with a message and a integer with the HTTP status code
         :rtype: Tuple(dict, integer)
         """
-        af_client = Airflow.from_config(current_app.config)
-        if not af_client.is_alive():
-            log.error("Airflow not accessible when getting schemas")
-            raise AirflowError(error="Airflow is not accessible")
+        user = Auth.get_user_obj_from_header(request.headers)
+        dags = PermissionsDAG.get_user_dag_permissions(user.id)
+        available_dags = [{"name": dag.dag_id, "test": 1} for dag in dags]
```
[the remark this answers is not in the document]
Comment (ggsdc) [reply cornflow/endpoints/schemas.py:37]: You are right, it should not appear, I do not know why it is there...

**Comment by ggsdc** [comment]
I won't merge this right away as tomorrow there is going to be a demo of cornflow, I do not want to face problems with this version.

**Pull request merged**